src/components/FormSteps/Profile.tsx

```diff
@@ -5,7 +5,10 @@ import useForm from "@hooks/useForm";
 
 const Profile = () => {
   const { updateForm, formData } = useForm();
-  const { firstName, lastName, email, password, repeatPassword } = formData;
+  const { firstName, lastName, email } = JSON.parse(
+    sessionStorage.getItem("signup_info") || "{}"
```

Comment:
deu um problema no build dizendo que "sessionStorage is not defined"

Reply:
mas o erro deu no register

Reply:
Fiz as alterações no register e no Profile também com o useEffect pra garantir que o next execute o sessionStorage com o client renderizado, podendo ter acesso ao objeto.

Reply:
![image](https://user-images.githubusercontent.com/57500163/233790216-d31fa3c5-4d0d-47df-a74c-22a0847a8c7e.png)
agora deu outro erro mano